docs/admin/administration.md

```diff
@@ -507,22 +502,38 @@ UUID, in addition to the rank state.
 
 When up and running, the entire system can be shutdown with the command:
 
+`$ dmg system stop [--force]`
+
+The output table will indicate action and result.
+
+While the engines are stopped, the DAOS servers will continue to
+operate and listen on the management network.
+
+!!! warning
+    All engines monitor each other and pro-actively exclude unresponsive
+    members. It is critical to properly stop a DAOS system as with dmg in
+    the case of a planned maintenance on all or a majority of the DAOS
+    storage nodes. An abrupt reboot of the storage nodes might result
+    in massive exclusion that will take time to recover.
+
+The force option can be passed to dmg system stop for cases when a clean
```

Comment:
maybe change to:
The force option can be passed to the dmg system stop when a clean shutown is not working. Monitoring is not disabled in this case, and spurious exclusion might happen, but the engines are guaranteed to be killed.